packages/backend-plugin-api/report.api.md

```diff
@@ -24,6 +24,40 @@ import { Readable } from 'stream';
 import type { Request as Request_2 } from 'express';
 import type { Response as Response_2 } from 'express';
 
+// @public (undocumented)
+export type AuditorCreateEvent<TRootMeta extends JsonObject> = (options: {
+  eventId: string;
+  severityLevel?: AuditorEventSeverityLevel;
+  request?: Request_2<any, any, any, any, any>;
+  meta?: TRootMeta;
+  suppressInitialEvent?: boolean;
+}) => Promise<{
+  success<TMeta extends JsonObject>(options?: { meta?: TMeta }): Promise<void>;
+  fail<TMeta extends JsonObject, TError extends Error>(
```

Comment:
There's no purpose to the type parameters, afaik they can be removed. Same with `TRootMeta`